qiskit/circuit/library/generalized_gates/gms.py

```diff
@@ -92,3 +94,24 @@ def __init__(self,
             for j in range(i + 1, self.num_qubits):
                 gms.append(RXXGate(theta[i][j]), [i, j])
         self.append(gms, self.qubits)
+
+
+class MSGate(Gate):
+    """MSGate has been deprecated.
+    Please use ``GMS`` in ``qiskit.circuit.generalized_gates`` instead.
```

Comment:
```suggestion
    """MSGate has been deprecated.
    Please use ``GMS`` in ``qiskit.circuit.generalized_gates`` instead.
    
    Global Mølmer–Sørensen gate.
    
    The Mølmer–Sørensen gate is native to ion-trap systems. The global MS can be
    applied to multiple ions to entangle multiple qubits simultaneously.
    
    In the two-qubit case, this is equivalent to an XX(theta) interaction,
    and is thus reduced to the RXXGate.
```

Reply:
Done in https://github.com/Qiskit/qiskit-terra/pull/4979/commits/29bbc333d01f453f93a4970271eec7091e2fe404